lib/GLPI/Agent/Task/Inventory/Generic/Dmidecode/Memory.pm

```diff
@@ -26,8 +26,8 @@ sub isEnabled {
 sub doInventory {
     my (%params) = @_;
 
-    my $inventory = $params{inventory};
-    my $logger    = $params{logger};
+    my $inventory      = $params{inventory};
+    my $logger         = $params{logger};
```

Comment:
You should also revert this space alignment change.

Reply:
Sorry for this omission. 😕